### DISCUSSION TITLE
Is there a arm64 package planned?

### DISCUSSION THREAD

**Discussion by bebewold** [rgriebl/brickstore#543] [Q&A] [answered]
I'm trying to install it on a arm64 linux computer from source code with ./configure && make && sudo make install but i keep running through error, its my first time installing from source code so i'm swimming blindly in all that and i was wondering if it'd be be betŧer for me to wait for the arm64 package to come out.

**Comment by rgriebl** [comment, marked as answer]
There is no single "arm64" Linux, so it's hard to provide a generic package. If you are on a Debian based Distro like ArmBIAN or RaspiOS, you just need to install the required Qt libraries first:

`$ sudo apt install libtbb-dev qtbase5-dev qt5-qmake qtbase5-dev-tools qtchooser qt5-style-plugins libqt5opengl5-dev qttools5-dev qtbase5-private-dev qtdeclarative5-dev`

Then a 
`$ ./configure && make`

should do the trick.
I've just tested this on a RPI4 and it worked just fine.

If you still get errors after installing the Qt libraries, then please attach the full output from the failed build here.

If this is for a Raspberry style setup: keep in mind, that BrickStore needs lots of RAM for all the item images: 2 GB really is a must have.

**Reply by bebewold** [reply]
I installed the libraries and it worked, thank you, I was probably only missing those.
And it's not a raspberry style setup, I was trying to install it on a Chromebook through Linux and my machine has a Snapdragon since it's designed more like a tablet. But everything works now thank you so much

**Reply by bebewold** [reply]
I'm trying to update the application but I'm missing some libraries again, can you tell me which libraries you used again?

**Reply by bebewold** [reply]
> I'm trying to update the application but I'm missing some libraries again, can you tell me which libraries you used again?

I'm missing all the qt6 libraries and apt can't find them for my distro... Is there an alternative I could use?

**Reply by rgriebl** [reply]
All your options are in Wiki: https://github.com/rgriebl/brickstore/wiki/Installation-Instructions
There's no way around using at least Qt 6.2 (which was released over one year ago)

**Comment by rgriebl** [comment]
Current BrickStore versions are built against Qt6, plus the build-system switched from using qmake to cmake. I've updated the build instructions in the Wiki though:
https://github.com/rgriebl/brickstore/wiki/Building-from-Source
